src/pages/BorrowerProfile/FundedBorrowerProfile.vue

```diff
@@ -111,6 +111,14 @@ export default {
 	metaInfo() {
 		const canonicalUrl = `https://${this.$appConfig.host}${this.$route.path}`.replace('funded', 'lend');
 		return {
+			title: `${this.loan?.name} From ${this.loan?.geocode?.country?.name}'s has been funded`,
```

Comment:
Do we want fallbacks if these `name` props or the `use` prop below are undefined?